nablapps/meeting_records/urls.py

```diff
@@ -6,7 +6,7 @@
 from .views import MeetingRecordDetailView, MeetingRecordListView
 
 urlpatterns = [
-    path("", MeetingRecordListView.as_view(), name="meeting_record_list"),
+    path("<int:year>/", MeetingRecordListView.as_view(), name="meeting_record_list"),
```

Comment:
En mulighet er å også legge inn en path på "", og så sende den til MeetingRecordListView, men med year som ekstra argument [(Django docs)](https://docs.djangoproject.com/en/3.1/topics/http/urls/#passing-extra-options-to-view-functions).